src/BloomExe/Publish/PublishView.cs

```diff
@@ -706,13 +706,7 @@ private void SetModelFromButtons()
 			}
 			if (pdfPreviewMode)
 			{
-				if (_model.DisplayMode == PublishModel.DisplayModes.Upload)
-				{
-					// We've transitioned away from upload to a PDF preview.
-					_model.DisplayMode = _model.PdfGenerationSucceeded
-						? PublishModel.DisplayModes.ShowPdf
-						: PublishModel.DisplayModes.WaitForUserToChooseSomething;
-				}
+				_model.DisplayMode = PublishModel.DisplayModes.Working;
```

Comment:
Friday pm again when I ask dumb questions...
Struggling to understand this change. How did it cause the bug? Why is this correct?

Reply:
I've revised the fix and added comments to explain what is going on here.